src/Databases/SQLite/SQLiteUtils.cpp

```diff
@@ -0,0 +1,57 @@
+#include "SQLiteUtils.h"
+
+#if USE_SQLITE
+#include <filesystem>
+
+namespace fs = std::filesystem;
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int PATH_ACCESS_DENIED;
+}
+
+
+String SQLiteDatabaseValidatePath(const String & path, const String & user_files_path)
```

Comment:
Consider to rename to validateSQLiteDatabasePath. For our style guide functions are not started with capital letter.